README.md

```diff
@@ -84,17 +86,17 @@ that describes it for both Unix and Windows systems.
 
 ## Signed Releases
 
-As of v0.10.15 and onwards release binaries are GPG signed with the key
-D26E6ED000654A3E, available from https://syncthing.net/security.html and
-most key servers.
+Beginning with `v0.10.15`, release binaries are GPG-signed with the key
+`D26E6ED000654A3E`, available from https://syncthing.net/security.html and
+most public key servers.
 
-There is also a built in automatic upgrade mechanism (disabled in some
-distribution channels) which uses a compiled in ECDSA signature. macOS
-binaries are also properly code signed.
+There is also a built-in, automatic-upgrade mechanism (disabled by default in 
```

Comment:
"automatic" here is an adjective describing the "upgrade" noun.  The hyphen is not needed.

The built-in upgrade is completely, forcefully disabled in some distribution channels.  Not only by default.  So this change is making it less accurate.